### PR TITLE
Claudio/examples

### PR DESCRIPTION
**Pull request by crusso**
Fixed the remaining examples that got away last time.

(manually type checked with moc - not checked for correctness though)

### CODE REVIEW

File: modules/developers-guide/examples/motoko-to-do/utils.mo [hunk 1]
```diff
@@ -2,52 +2,59 @@
...
-		};
-	};
-	updated
-};
+// Import a local library for the ToDO type definition
+import Types "./types";
+
+module {
+
+
```
Comment (crusso) [inline modules/developers-guide/examples/motoko-to-do/utils.mo:10]:
```suggestion
```

File: modules/developers-guide/examples/motoko-to-do/types.mo [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+module { 
+  
+  // ToDo item properties
+  public type ToDo = {
+    id : Nat;
+    description : Text;
+    completed : Bool;
+  };
+
+}
```
Comment (crusso) [inline modules/developers-guide/examples/motoko-to-do/types.mo:10]:
```suggestion
}

```

Comment (crusso) [inline modules/developers-guide/examples/motoko-to-do/types.mo]:
```suggestion
}

```

**Review by lsgunnlsgunn — APPROVED** [review]
stamped!

**Pull request merged**